tool-plugins/intellij/README.md

```diff
@@ -1,57 +1,89 @@
-
-**Note:** On Windows, you don’t need the leading `./` in front of the `gradlew` command
-
-## Installing the plugin to IDEA
+[Ballerina](https://ballerina.io/) is a simple programming language whose syntax and runtime address the hard 
+problems of integration. As a Turing complete language, Ballerina enables agility with edit, build, and run cycles. 
+Ballerina code is compiled into services that include transactions, embedded brokers and gateway runtime.
+Ballerina makes it easy to build resilient services that integration and orchestrate across distributed endpoints. 
+
+Ballerina plugin provides Ballerina Language support in IntelliJ IDEA .
+
+## Table of Contents
+
+- [**Getting Started**](getting-started/README.md)
+    - [How to Install](#how-to-install)
+    - [Setting Up Ballerina SDK](getting-started/setting-up-ballerina-sdk/README.md)
+    - [Writing Hello World In Ballerina](getting-started/writing-hello-world/README.md)
+    
+- [**Plugin Features**](getting-started/plugin-features/README.md)
+    - [Running Ballerina Programs](getting-started/plugin-features/README.md#running-ballerina-programs)
+    - [Debugging Ballerina Programs](getting-started/plugin-features/README.md#debugging-ballerina-programs) 
+    - [Import modules On-the-fly](getting-started/plugin-features/README.md#import-modules-on-the-fly)
+    - [Auto Import Unambiguous Modules](getting-started/plugin-features/README.md#auto-import-unambiguous-modules)
+    - [Find Usage](getting-started/plugin-features/README.md#find-usage)
+    - [Go to Definition](getting-started/plugin-features/README.md#go-to-definition)
+    - [Formatting](getting-started/plugin-features/README.md#formatting)
+    - [Parameter Information](getting-started/plugin-features/README.md#parameter-information)
+    - [Quick Documentation](getting-started/plugin-features/README.md#quick-documentation)
+    - [Object/Record Fields suggestions](getting-started/plugin-features/README.md#objectrecord-fields-suggestions)
+    - [Annotation fields suggestions](getting-started/plugin-features/README.md#annotation-fields-suggestions)
+    - [File Templates](getting-started/plugin-features/README.md#file-templates)
+    - [Live Templates and Code snippets](getting-started/plugin-features/README.md#live-templates-and-code-snippets)
+    - [Spell Checking](getting-started/plugin-features/README.md#spell-checking)
+    - [Semantic Analyzing](getting-started/plugin-features/README.md#semantic-analyzing) 
+
+- [**Plugin Developer Guide**](getting-started/plugin-developer-guide/README.md#plugin-developer-guide)
+    - [Testing/Debugging the plugin using IntelliJ](getting-started/plugin-developer-guide/README.md#testingdebugging-the-plugin-using-intellij-idea)
+    
+- [**Release versions schema**](#release-versions-schema)
+- [**How to Contribute**](#how-to-contribute) 
+- [**Licence**](#license)
+- [**Copyright**](#copyright)
+- [**Useful Links**](#useful-links)
+
+
+## How to Install
```

Comment:
There is some inconsistency with titles. Some are title case, and some are not. I think we usually go with title case everywhere for titles so please apply throughout the document/